recipes/terraform-provider-alicloud/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set provider= "alicloud" %}
+{% set version = "1.8.0" %}
+
+{% set name = "terraform-provider-" + provider %}
+{% set x4_plugin = name + '_v' + version + "_x4" %}
+
+{% set sha256 = "265c79575f61191966a43c58043f269f3ceb200546a8607ac1c85c6e75007e59" %}
+{% set sha256_win32 = "6ae2b2ec933fc924c16b88ba446f18356805f7f987ef7d35146c7dbc82505f83" %}
+{% set sha256_win64 = "b2cbe6d26af42c7cd52145732c95d0736c1c5b8207350e40de9b4701a20b11d5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/terraform-providers/{{ name }}/archive/v{{ version }}.tar.gz  # [not win]
+  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
+  sha256: {{ sha256 }}  # [not win]
+
+  url: https://releases.hashicorp.com/{{ name }}/{{ version }}/{{ name }}_{{ version }}_windows_386.zip  # [win32]
+  url: https://releases.hashicorp.com/{{ name }}/{{ version }}/{{ name }}_{{ version }}_windows_amd64.zip  # [win64]
+  fn: {{ name }}-{{ version }}.zip  # [win]
+  sha256: {{ sha256_win32 }}  # [win32]
+  sha256: {{ sha256_win64 }}  # [win64]
+
+build:
+  number: 0
+  script:
+    - XCOPY {{ x4_plugin }}.exe %LIBRARY_BIN%  # [win]
```

Comment:
Should the exe be versioned?

Reply:
the short answer is yes. I will follow up with the link to the upstream documentation in the next post.